test/integration/buildpod_admission_test.go

```diff
@@ -107,6 +108,40 @@ func TestBuildDefaultAnnotations(t *testing.T) {
 	}
 }
 
+func TestBuildOverrideTolerations(t *testing.T) {
+	tolerations := []kapi.Toleration{
+		{
+			Key:      "mykey1",
+			Value:    "myvalue1",
+			Effect:   "NoSchedule",
+			Operator: "Equal",
+		},
+		{
+			Key:      "mykey2",
+			Value:    "myvalue2",
+			Effect:   "NoSchedule",
+			Operator: "Equal",
+		},
+	}
+
+	oclient, kclientset, fn := setupBuildOverridesAdmissionTest(t, &overridesapi.BuildOverridesConfig{
+		Tolerations: tolerations,
+	})
+
+	defer fn()
+
+	_, pod := runBuildPodAdmissionTest(t, oclient, kclientset, buildPodAdmissionTestDockerBuild())
+	for i, toleration := range tolerations {
+		tol := v1.Toleration{}
+		if err := kapiv1.Convert_core_Toleration_To_v1_Toleration(&toleration, &tol, nil); err != nil {
+			panic(err)
```

Comment:
while you "can" panic here, it's not going to make it easier to investigate the failure.  just fail the test w/ the error.

Reply:
updated